pkg/action/install.go

```diff
@@ -363,12 +366,56 @@ func (c *Configuration) renderResources(ch *chart.Chart, values chartutil.Values
 
 	// Aggregate all valid manifests into one big doc.
 	for _, m := range manifests {
-		fmt.Fprintf(b, "---\n# Source: %s\n%s\n", m.Name, m.Content)
+		if outputDir == "" {
```

Comment:
Should it be checked here for a blank template? This was similar to how it was checked in v2:
```console
if whitespaceRegex.MatchString(m.Content) {
        continue
}
```